geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messages/AbstractGMSMessage.java

```diff
@@ -14,15 +14,16 @@
  */
 package org.apache.geode.distributed.internal.membership.gms.messages;
 
+import java.util.ArrayList;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 
 import org.apache.geode.annotations.Immutable;
 import org.apache.geode.distributed.internal.membership.gms.api.MemberIdentifier;
-import org.apache.geode.distributed.internal.membership.gms.interfaces.GMSMessage;
-import org.apache.geode.internal.serialization.DataSerializableFixedID;
+import org.apache.geode.distributed.internal.membership.gms.api.Message;
 
-public abstract class AbstractGMSMessage implements DataSerializableFixedID, GMSMessage {
+public abstract class AbstractGMSMessage implements Message {
   @Immutable
   public static final MemberIdentifier ALL_RECIPIENTS = null;
```

Comment:
`ALL_RECIPIENTS` should not be defined in this class because it's already defined up in the `Message` interface

Reply:
@Bill maybe make a ticket to improve this separate from this PR

Reply:
I missed that one - I'll address it in this ticket.

Reply:
oh - I had to keep that one because the two ALL_RECIPIENTS have different types.